routers/web/repo/pull.go

```diff
@@ -1500,8 +1500,8 @@ func CompareAndPullRequestPost(ctx *context.Context) {
 				ctx.ServerError("CompareAndPullRequest.HTMLString", err)
 				return
 			}
-			ctx.Flash.Error(flashError)
-			ctx.JSONRedirect(pullIssue.Link()) // FIXME: it's unfriendly, and will make the content lost
+			ctx.Flash.Warning(flashError)
+			ctx.JSONRedirect(ctx.Link)
```

Comment:
Well, the code just went back to the old. So why need this PR then?

![image](https://github.com/go-gitea/gitea/assets/2114189/021e183b-9aa5-4207-80b2-fa426abe0cd3)

Reply:
But why not keep the "Error" and keep the comment?

Reply:
The UI very special, when using Error, the error prompt section will displayed on the left of the input box.

Reply:
So when developing Gitea, when there is a unclear problem, it should always be hacky patched, but not really resolve it clearly?